scripts/skills/perks/perk_rf_unstoppable.nut

```diff
@@ -30,95 +27,101 @@ this.perk_rf_unstoppable <- ::inherit("scripts/skills/skill", {
 		return this.m.Stacks == 0;
 	}
 
+	function getAPBonus()
+	{
+		return this.m.Stacks;
+	}
+
+	function getInitiativeBonus()
+	{
+		return this.m.Stacks * 10;
+	}
+
 	function getTooltip()
 	{
 		local tooltip = this.skill.getTooltip();
 
-		tooltip.push({
-			id = 10,
-			type = "text",
-			icon = "ui/icons/hitchance.png",
-			text = "[color=" + ::Const.UI.Color.PositiveValue + "]+" + this.getSkillBonus() + "[/color] Melee Skill"
-		});
-
-		local APBonus = this.getAPBonus();
-		if (APBonus > 0)
+		if (this.getAPBonus() != 0)
 		{
 			tooltip.push({
 				id = 10,
 				type = "text",
 				icon = "ui/icons/action_points.png",
-				text = "[color=" + ::Const.UI.Color.PositiveValue + "]+" + APBonus + "[/color] Action Point(s)"
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeValue(this.getAPBonus()) + " [Action Points|Concept.ActionPoints]")
 			});
 		}
 
-		return tooltip;
-	}
-
-	function onBeforeTargetHit( _skill, _targetEntity, _hitInfo )
-	{
-		this.m.Distance = 0;
-		this.m.APBonusBefore = this.getAPBonus();
-
-		if (_skill.isAttack() && !_targetEntity.isAlliedWith(this.getContainer().getActor()))
+		if (this.getInitiativeBonus() != 0)
 		{
-			this.m.Distance = _targetEntity.getTile().getDistanceTo(this.getContainer().getActor().getTile());
+			tooltip.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/initiative.png",
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeValue(this.getInitiativeBonus()) + " [Initiative|Concept.Initiative]")
+			});
 		}
-	}
 
-	function onTargetHit( _skill, _targetEntity, _bodyPart, _damageInflictedHitpoints, _damageInflictedArmor )
-	{
-		local actor = this.getContainer().getActor();
-		if (_skill.isAttack() && ::Tactical.TurnSequenceBar.isActiveEntity(actor) &&!_targetEntity.isAlliedWith(actor))
-		{
-			this.m.Stacks = ::Math.minf(this.m.MaxStacks, this.m.Stacks + (this.m.Distance > 1 ? 0.5 : 1));
-			actor.setActionPoints(actor.getActionPoints() + this.getAPBonus() - this.m.APBonusBefore);
-		}
-	}
+		tooltip.push({
+			id = 11,
```

Comment:
Now the id 11 is used twice.
This should be a unique id, e.g. 12 (if 12 is indeed unused by the time you implement this change)